app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -31,7 +31,7 @@ def import_vm(source_vm_id, target_params)
   end
 
   def validate_import_vm
-    api_version >= '4.0'
+    highest_supported_api_version && highest_supported_api_version >= '4'
```

Comment:
Hmm, ok I just wonder why [this](https://github.com/ManageIQ/manageiq-providers-ovirt/blob/d401d4cb4276143c4e6ecce00e5bae4aa59b8c47/app/models/manageiq/providers/redhat/infra_manager/api_integration.rb#L237) returns `Integer(3)` ...